src/ucp/core/ucp_worker.c

```diff
@@ -202,6 +202,18 @@ static void ucp_worker_set_config(ucp_worker_h worker, ucp_rsc_index_t tl_id)
     config->rndv_thresh  = SIZE_MAX;
 }
 
+static void ucp_worker_set_stub_config(ucp_worker_h worker)
+{
+    ucp_context_h context        = worker->context;
+    ucp_ep_config_t *config      = &worker->ep_config[context->num_tls];
+
+    memset(config, 0, sizeof(*config));
+
+    config->max_am_bcopy      = 256;
```

Comment:
needed some minimal value. this is something temporary i indent to fix very soon.